src/server/middleware.ts

```diff
@@ -22,19 +22,25 @@ export function setDefaultCsp({
 // interval is rather arbitrary and could be set higher (less server load) or lower (fresher data).
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
-export function setCacheControl({
-  res,
-  next,
-}: {
-  res: Response;
-  next: NextFunction;
-}) {
+export function setCacheControl(
+  req: Request,
+  res: Response,
+  next: NextFunction
+) {
   const user = UserService.Instance;
   let caching: string;
-  if (user.auth()) {
-    caching = "private";
+  if (
+    req.path.match(/\.(js|css|txt|manifest\.webmanifest)\/?$/) ||
+    req.path.includes("/css/themelist")
+  ) {
+    // Static content gets cached publicly for a day
+    caching = "public, max-age=86400";
   } else {
-    caching = "public, max-age=60";
+    if (user.auth()) {
+      caching = "private";
+    } else {
+      caching = "public, max-age=60";
```

Comment:
Would changing `max-age` here to `5` fix/mitigate our captcha and logout issues?

Reply:
Not sure. I left a comment on your other PR about the [Clear-Site-Data](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data) header. That would require a change to the js client instead of the UI, but it could help. The only rub is that the `cache` option of site data is not supported in Firefox.